apps/hpc-ftsadmin/src/i18n/langs/en.json

```
@@ -44,15 +51,215 @@
       "error": "Error:",
       "retry": "Retry"
     },
+    "reviewPendingValues": {
+      "unmatched": "Unmatched",
+      "label": "Change: New Value",
+      "button": {
+        "accept": "Accept",
+        "reject": "Reject",
+        "confirm": "Confirm handled case"
+      }
+    },
     "unsupportedBrowser": {
       "title": "Unsupported Browser!",
       "info": "Unfortunately, your current browser is not supported. Please switch to either Firefox or Google Chrome. To do this, simply copy the form address (URL) above, then open Firefox or Chrome, and paste the address into the browser."
     },
+    "flow": {
+      "updatedBy": "Updated {date} by {user}",
+      "createdBy": "Created {date} by {user}",
+      "deleted": "This flow has been deleted",
+      "inactiveReason": "This flow is not active because it has been marked as {reason}",
+      "unknownInactiveReasons": "Unknown inactive reasons",
```

I think this should be in singular form. We should only have a single inactive reason per flow version, even though there is some corrupt data in the DB.
```suggestion
      "unknownInactiveReasons": "Unknown inactive reason",
```